https://bugs.gentoo.org/33315
glibc 2.2.x does not define LOGIN_NAME_MAX
	(12 Nov 2003) -taviso@gentoo.org

--- skey-1.1.5.orig/skeyinit.c
+++ skey-1.1.5/skeyinit.c
@@ -62,6 +62,11 @@
 #define SKEY_NAMELEN    4
 #endif
 
+/* #33315 */
+#ifndef LOGIN_NAME_MAX
+#define LOGIN_NAME_MAX 256
+#endif
+
 int	main __P((int, char **));
 
 int main(int argc, char **argv)
